api-server/scanner-lib/src/blockchain_state/mod.rs

```diff
@@ -1075,12 +1075,25 @@ async fn update_tables_from_transaction_outputs<T: ApiServerStorageWrite>(
                 };
                 db_tx.set_fungible_token_issuance(token_id, block_height, issuance).await?;
             }
-            TxOutput::IssueNft(token_id, issuance, _) => {
+            TxOutput::IssueNft(token_id, issuance, destination) => {
+                let address = Address::<Destination>::new(&chain_config, destination.clone())
+                    .expect("Unable to encode destination");
+                address_transactions.entry(address.clone()).or_default().insert(transaction_id);
+
+                increase_address_amount(
+                    db_tx,
+                    &address,
+                    &Amount::from_atoms(1),
+                    CoinOrTokenId::TokenId(*token_id),
+                    block_height,
+                )
+                .await;
+
                 db_tx.set_nft_token_issuance(*token_id, block_height, *issuance.clone()).await?;
                 set_utxo(
                     outpoint,
                     output,
-                    Some(1),
```

Comment:
I believe this needs to stay, otherwise when fetching the utxos it will not know how many decimals to use for formatting the amount.

Reply:
nope, seems it is all good